src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -984,3 +985,12 @@ def list_revision(cmd,
         return retrieved_revisions
     except HttpResponseError as ex:
         raise CLIErrors.AzureResponseError('List revision operation failed.\n' + str(ex))
+
+
+# Helper function to validate JSON input that may contain comments
+def __validate_json_input(json_string):
```

Comment:
If you ignore my comment above, couldn't we make this the same method as the one above and move it to utilities.

Reply:
Updated. Moved to `_json` file